doc/dev-guide/src/index.md

```diff
@@ -3,7 +3,7 @@
 1. Fork it!
 2. Create your feature branch: `git checkout -b my-new-feature`
 3. Test it: `cargo test --features=test`
-4. Lint it: `cargo +beta clippy --all --all-targets -- -D warnings`
+4. Lint it: `cargo +beta clippy --all --all-targets --all-features -- -D warnings`
```

Comment:
We would like to prevent unnecessary duplication here.
```suggestion
4. [Lint it!](linting.md)
```

Reply:
Thank you!
How did you add recommendation that could be commited directly?

Reply:
@vpochapuis

Via https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request